File: masar_soqoor/custom/journal_entry/journal_entry.js
frappe.ui.form.on("Journal Entry Account", "account", function(frm, cdt, cdn) {
    var d = locals[cdt][cdn];
    if (d.account === "215001 - مصاريف رواتب  مستحقة الدفع - SATC") {
        frappe.model.set_value(cdt, cdn, "party_type", "Employee");
    } else if (d.account === "52110001 - رواتب و اجور إداري - SATC") {
        frappe.model.set_value(cdt, cdn, "party_type", "Employee");
    } else if (d.account === "5220001 - رواتب واجور قسم المبيعات - SATC") {
        frappe.model.set_value(cdt, cdn, "party_type", "Employee");
    } else if (d.account === "5120001 - رواتب و اجور انتاج - SATC") {
        frappe.model.set_value(cdt, cdn, "party_type", "Employee");
    } else if (d.account === "114206 - ذمم الموظفين - SATC") {
        frappe.model.set_value(cdt, cdn, "party_type", "Employee");
    }
    refresh_field("party_type", cdt, cdn);
});

File: masar_soqoor/custom/journal_entry/journal_entry.py
import frappe

@frappe.whitelist()
def update_employee(doc, method):
    for account in doc.accounts:
        if account.account == "215001 - مصاريف رواتب  مستحقة الدفع - SATC":
            name = frappe.db.get_value("Employee", account.party, "name")
            if name:
                account.employee = name
        elif account.account == "52110001 - رواتب و اجور إداري - SATC":
            name = frappe.db.get_value("Employee", account.party, "name")
            if name:
                account.employee = name
        elif account.account == "5220001 - رواتب واجور قسم المبيعات - SATC":
            name = frappe.db.get_value("Employee", account.party, "name")
            if name:
                account.employee = name
        elif account.account == "5120001 - رواتب و اجور انتاج - SATC":
            name = frappe.db.get_value("Employee", account.party, "name")
            if name:
                account.employee = name
        elif account.account == "114206 - ذمم الموظفين - SATC":
            name = frappe.db.get_value("Employee", account.party, "name")
            if name:
                account.employee = name
